qiskit_experiments/framework/experiment_data.py

```diff
@@ -1513,7 +1513,9 @@ def save(
         for data in self._child_data.values():
             original_verbose = data.verbose
             data.verbose = False
-            data.save()
+            data.save(
+                suppress_errors=suppress_errors, max_workers=max_workers, save_figures=save_figures
```

Comment:
If we have 20 sub-experiments, and `max_workers` is set to 10, then will we try to apply 200 workers? Any way to avoid it?

Reply:
Good point. No way to avoid it by sharing the workers since they are created in `qiskit-ibm-experiment`, but since currently save is blocking, saving the sub-experiments will be done sequentially and so no more than `max_workers` threads will be used.

If we want to change the save to be non-blocking, this is one of the many problems we'll face.